### ISSUE TITLE
Pagetoid Proliferation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9951]
**Preferred term label:**
Pagetoid Proliferation

**Synonyms**
Skip lesions, non-cohesive spread, spotty spread, eyelid/conjunctival involvement, intraepidermal spread

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C1335292

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sebaceous cell carcinoma of the eyelid

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I think it is probably better to make a term "Sebaceous Carcinoma of the Eyelid", because the above is one of the findings of that disease? 

https://pubmed.ncbi.nlm.nih.gov/33347004/

**Comment by MickeySegal** [comment]
The point of having a clinical finding is that it can be observed by a clinician without doing a lab test.  substituting a pathological diagnosis loses the value of that clinical observation.
https://eyewiki.org/Sebaceous_Carcinoma
"One of the hallmark features of sebaceous carcinoma is pagetoid spread--spreading of malignant cells to epithelium that appears to be separate from main tumor. "

**Comment by pnrobinson** [comment]
@MickeySegal 
yes but spreading of "malignant" cells means it is a lab test? I am not sure what the actual phenotypic observation is?

**Comment by MickeySegal** [comment]
The middle of the first set of 3 images shows the meaning macroscopically - I would call it skip lesions, but apparently Pagetoid spread is the term used by ophthalmologists.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001204
- label: Pagetoid spread
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: PMID:20431478
- def: Pagetoid spread connotes the spread of single neoplastic cells, and small nests of them above the basal layer in a variety of melanocytic and nonmelanocytic neoplasms similar to the pattern seen in Paget's and in extramammary Paget disease.
- comment: Pagetoid proliferation of single melanocytic cells or small nests of melanocytes may be seen in a variety of melanocytic neoplasms including pagetoid spitz nevi, de novo epithelioid melanocytic dysplasia, and melanoma. Pagetoid melanocytosis may also be a common feature of acral nevi, traumatized nevi, or in nevi after acute sun exposure.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9951
- synonyms:
